MMA Removal Utility/SingleTenantSetupInstallation.md

```diff
@@ -0,0 +1,302 @@
+# **Steps to install single-tenant AzTS MMA Removal Utility**
+
+In this section, we will walk you through the steps of setting up AzTS MMA Removal Utility. This setup can take up to 30 minutes.
+
+Before initiating the setup please validate the prerequisites, download and extract deployment package using [Prerequisites](./Prerequisites.md).
+
+There are two methods to setup AzTS MMA Removal Utility solution:
+
+- [**Method A:**](#method-a) This method provides granular level details of the different steps involved in setting up AzTS MMA Removal Utility. Setup process is divided into five steps and seprate command is provided for each step.
+
+- [**Method B:**](#method-b) Provides a quick way to install AzTS MMA Removal Utility solution by wrapping up multiple steps into a single consolidated command. 
+____________________________________________________________________
+
+## **Method A:**
+
+> _**Note:** Please validate the prerequisites [here](./Prerequisites.md). You can download the deployment package zip from [here](https://github.com/azsk/AzTS-docs/raw/main/TemplateFiles/AzTSMMARemovalUtilityDeploymentFiles.zip) and before extracting the zip file, right click on the zip file --> click on 'Properties' --> Under the General tab in the dialog box, select the 'Unblock' checkbox --> Click on 'OK' button. Extract the zip file and use **MMARemovalUtilitySetup.ps1** present in this package to run the commands mentioned in below steps_
```

Comment:
Note seems to be redundant

Reply:
Simplified the note, (retaining the note as there is a chance users might miss it though mentioned at the beginning of file) thoughts?